src/main/java/com/bina/varsim/util/VCFparser.java

```diff
@@ -319,301 +319,286 @@ else if (index == 9) { // Output format
+
+              if (svlen.length > 0) {
+                  //0 is for reference allele
+                  //alternative allele is numbered 1,2,... per VCFSpec
+                  for (int altAlleleIndex = 1; altAlleleIndex <= svlen.length; altAlleleIndex++) {
+                      int currentCopyNumber = 1;
                     /*
                     implicit assumption here: genotype[0] == genotype[1] => copyNumberArray[0] == copyNumberArray[1]
                      */
-                    //check paternal
-                    if (altAlleleIndex == genotypeArray[0]) {
-                        currentCopyNumber = copyNumberArray[0];
-                    }
-                    //check maternal
-                    if (altAlleleIndex == genotypeArray[1]) {
-                        currentCopyNumber = copyNumberArray[1];
-                    }
-                    currentCopyNumber = Math.max(1, currentCopyNumber);
-                    int altAllelelength = Math.max(Math.abs(svlen[altAlleleIndex - 1]), 1);
+                      //check paternal
+                      if (altAlleleIndex == genotypeArray[0]) {
+                          currentCopyNumber = copyNumberArray[0];
+                      }
+                      //check maternal
+                      if (altAlleleIndex == genotypeArray[1]) {
+                          currentCopyNumber = copyNumberArray[1];
+                      }
+                      currentCopyNumber = Math.max(1, currentCopyNumber);
+                      int altAllelelength = Math.max(Math.abs(svlen[altAlleleIndex - 1]), 1);
                     /*Not sure if INS is the most appropriate FlexSeq.Type
                     use it unless it breaks things.
                      */
-                    if (subtype[altAlleleIndex - 1].equals("ACCEPT")) {
-                        alts[altAlleleIndex - 1] = new FlexSeq(FlexSeq.Type.TRA, altAllelelength, currentCopyNumber);
-                    } else if (subtype[altAlleleIndex - 1].equals("REJECT")){
-                        alts[altAlleleIndex - 1] = new FlexSeq(FlexSeq.Type.DEL, 0);
-                    } else {
-                        throw new IllegalArgumentException("ERROR: only ACCEPT and REJECT allowed.\n" + line);
-                    }
-                }
-
-                //TODO: there could be multiple SVLEN values, but for now we only use one
-                //make sure all SVLEN values are equal if we only use the first one
-                //per VCFv4.1 spec, SVLEN should be length of alternative allele rather than
-                //reference allele
-                for (int i = 1; i < svlen.length; i++) {
-                    if (svlen[i] != svlen[0]) {
-                        throw new IllegalArgumentException("ERROR: SVLEN values not equal.\n" + line);
-                    }
-                }
-                //pos is incremented by 1, so it becomes 1-based start
+                      if (subtype[altAlleleIndex - 1].equals("ACCEPT")) {
```

Comment:
Should make constant strings like these as constant variables defined in the class.